truffle/src/main/java/org/truffleruby/options/CommandLineParser.java

```diff
@@ -600,36 +563,89 @@ private String grabOptionalValue() {
+        out.println("  -Eex[:in], --encoding=ex[:in]");
+        out.println("                  specify the default external and internal character encodings");
+        out.println("  -Fpattern       split() pattern for autosplit (-a)");
+        out.println("  -i[extension]   edit ARGV files in place (make backup if extension supplied)");
+        out.println("  -Idirectory     specify $LOAD_PATH directory (may be used more than once)");
+        out.println("  -l              enable line ending processing");
+        out.println("  -n              assume 'while gets(); ... end' loop around your script");
+        out.println("  -p              assume loop like -n but print line also like sed");
+        out.println("  -rlibrary       require the library before executing your script");
+        out.println("  -s              enable some switch parsing for switches after script name");
+        out.println("  -S              look for the script using PATH environment variable");
+        out.println("  -T[level=1]     turn on tainting checks");
+        out.println("  -v, --verbose   print version number, then turn on verbose mode");
+        out.println("  -w              turn warnings on for your script");
+        out.println("  -W[level=2]     set warning level; 0=silence, 1=medium, 2=verbose");
+        out.println("  -x[directory]   strip off text before #!ruby line and perhaps cd to directory");
+        out.println("  --copyright     print the copyright");
+        out.println("  --enable=feature[,...], --disable=feature[,...]");
+        out.println("                  enable or disable features");
+        out.println("  --external-encoding=encoding, --internal-encoding=encoding");
+        out.println("                  specify the default external or internal character encoding");
+        out.println("  --version       print the version");
+        out.println("  --help          show this message, -h for short message");
+        out.println("Features:");
+        out.println("  gems            rubygems (default: enabled)");
+        out.println("  did_you_mean    did_you_mean (default: enabled)");
+        out.println("  rubyopt         RUBYOPT environment variable (default: enabled)");
+        out.println("  frozen-string-literal");
+        out.println("                  freeze all string literals (default: disabled)");
+        out.println("TruffleRuby:");
+        out.println("  -Xlog=severe,warning,performance,info,config,fine,finer,finest");
+        out.println("                  set the TruffleRuby logging level");
+        out.println("  -Xoptions       print available TrufleRuby options");
+        out.println("  -Xname=value    set a TruffleRuby option (omit value to set to true)");
+
+        if (TruffleOptions.AOT) {
+            out.println("SVM:");
+            out.println("  -Dname=value     set a system property");
+            out.println("  -J:arg, -J-arg  pass arg to the JVM");
```

Comment:
I think it would be great to have a single unified way to pass Java options, no matter SVM or HotSpot.